tests/unit/expire.tcl

```diff
@@ -571,6 +571,18 @@ start_server {tags {"expire"}} {
                 assert_equal [$primary pexpiretime $key] [$replica pexpiretime $key]
             }
         }
+
+        test {expired key which is created in writeable replicas should be deleted by active expiry} {
+            $primary flushall
+            $replica config set replica-read-only no
+            foreach {yes_or_no} {yes no} {
+                $replica config set appendonly $yes_or_no
+                $replica set foo bar PX 1
+                #wait bigger than 100ms to make sure that key has been deleted by active expiry
+                after 150
```

Comment:
please use
```
set prev_expired [s expired_keys]
$replica set foo bar PX 1
wait_for_condition 100 10 {
    [s expired_keys] eq $prev_expired + 1
} else {
    fail "key not expired"
}
```

Reply:
Thanks for your patience, this way is indeed more accurate and faster